runtime/amplitude/src/lib.rs

```diff
@@ -1071,11 +1072,35 @@ impl staking::Config for Runtime {
 	type CurrencyId = CurrencyId;
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+pub struct DataFeederBenchmark<K, V, A>(PhantomData<(K, V, A)>);
+
+#[cfg(feature = "runtime-benchmarks")]
+impl<K, V, A> orml_traits::DataFeeder<K, V, A> for DataFeederBenchmark<K, V, A> {
+	fn feed_value(_who: A, _key: K, _value: V) -> sp_runtime::DispatchResult {
+		Ok(())
+	}
+}
+
+#[cfg(feature = "runtime-benchmarks")]
+impl<K, V, A> orml_traits::DataProvider<K, V> for DataFeederBenchmark<K, V, A> {
+	fn get(_key: &K) -> Option<V> {
+		None
+	}
+}
+
 impl oracle::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type WeightInfo = oracle::SubstrateWeight<Runtime>;
 	type DataProvider = DataProviderImpl;
+	#[cfg(feature = "runtime-benchmarks")]
+	type DataFeedProvider = DataFeederBenchmark<
+		oracle::OracleKey,
+		oracle::TimestampedValue<UnsignedFixedPoint, Moment>,
+		Self::AccountId,
+	>;
```

Comment:
What's that doing?

Reply:
to run the benchmark, the `type DataFeedProvider` has to be filled. This is used solely for spacewalk's integration tests. I would have implemented `DataFeeder` and `DataProvider` traits for `()`, however both traits have generics, so I created a dummy struct `DataFeederBenchmark`.

Reply:
Okay, sounds good 👍